.travis.yml

```diff
@@ -5,8 +5,21 @@ python:
     - '3.6'
 install:
     - make install-dev
-    - pip install codecov
+    - pip install codecov httpie
+    - wget -O oc.tar.gz https://github.com/openshift/origin/releases/download/v3.7.2/openshift-origin-client-tools-v3.7.2-282e43f-linux-64bit.tar.gz
+    - tar -zxvf oc.tar.gz
+    - cp openshift-origin-client-tools-v3.7.2-282e43f-linux-64bit/oc .
+before_script:
+    - ./oc login $OPENSHIFT_URL --token=$OPENSHIFT_TOKEN
+    - REVISION=$(./oc rollout latest dc/postgresql -o revision)
+    - ./oc rollout status dc/postgresql --revision=$REVISION
+    - REVISION=$(./oc rollout latest dc/cloudigrade -o revision)
+    - ./oc rollout status dc/cloudigrade --revision=$REVISION
+    - ./oc rsh -c cloudigrade $(./oc get pods -o jsonpath='{.items[*].metadata.name}' -l name=cloudigrade) scl enable rh-postgresql96 rh-python36 -- python manage.py createsuperuser --no-input --username $CLOUDIGRADE_USER --email="$CLOUDIGRADE_USER@example.com"
+    - CLOUDIGRADE_TOKEN=$(./oc rsh -c cloudigrade $(./oc get pods -o jsonpath='{.items[*].metadata.name}' -l name=cloudigrade) scl enable rh-postgresql96 rh-python36 -- python manage.py drf_create_token $CLOUDIGRADE_USER | awk '{print $3}')
+    - AUTH=Authorization:"Token $CLOUDIGRADE_TOKEN"
```

Comment:
Can you 1) stick this in a script and source it here (or put in integrade Make file) 2) also set these variables: 

$CLOUDIGRADE_BASE_URL= what is the route to cloudigrade (just the base url, not /api/v1.(domain name only, not http/https)
$USE_HTTPS=boolean (True/False) --> I will assume false, so no need to set it if it is served over http
$SSL_VERIFY=boolean(True/False) --> I will assume false, so no need to set it if insecure